src/content/en/updates/_shared/deprecations.html

```diff
@@ -0,0 +1 @@
+New deprecations for this version of Chrome are listed below. Chrome Platform Status provides a [list of deprecated features](https://www.chromestatus.com/features#browsers.chrome.status:%22Deprecated%22) from this and previous versions of Chrome.
```

Comment:
You can't use markdown in HTML files, please change the markdown link to an `<a href...` link.